src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -62,7 +62,7 @@ export const board = (formValues, noDRB) => {
 
     if (formValues['1_branchOfService'] === 'airForce') {
       return {
-        name: 'Air Force Review Boards Agency (AFDRB) for the Air Force',
+        name: 'Air Force Review Boards Agency (AFDRB)',
```

Comment:
I asked Fran about removing this redundancy when I made a content change in May. I don't think we got confirmation that it should be removed.

Reply:
@FranECross Wondering if this redundancy can be removed? On prod it reads  `Air Force Review Boards Agency (AFDRB) for the Air Force for the Air Force`

Reply:
Oh I didn't realize the last four words were actually repeated. I just thought it said `Air Force Review Boards Agency (AFDRB) for the Air Force` (which is redundant too)